backend/ianalyzer/settings.py

```diff
@@ -48,7 +48,7 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 CSRF_TRUSTED_ORIGINS = ['http://localhost:8000']
 
-SITE_NAME = 'IANALYZER'
+SITE_NAME = 'Textcavator'
```

Comment:
This change requires a data migration, or manual change in the admin interface. See de7ea388014d4354de4788680f1f375ae1979bfc